### ISSUE TITLE
Slashes inside backticks are interpreted as comment

### ISSUE THREAD

**Issue by SimonRask** [uben0/tree-sitter-typst#10]
Example:

```typ
a
`//`
b
`c`
```

Actual:
```
(source_file
	(text)
	(raw_span
		(comment)
		(blob))
	(text)
	(ERROR
		(blob)))
```

Expected:

```
(source_file
	(text)
	(raw_span
		(blob))
	(text)
	(raw_span
		(blob)))
```

**Comment by uben0** [comment]
I fixed it in 60843a0604120c3014507f9635300264ba0e547e
Can you try and see if you can still find a buggy situation?

**Comment by SimonRask** [comment]
Thanks, it fixed my issue 🎉
